src/main/webapp/populateData.js

```diff
@@ -337,6 +337,22 @@ function populatePlanMail() {
       });
 }
 
+/**
+ * Call a post request to create a new event in the calendar, then display the
+ * updated information in calendar and plan panels
+ *
+ * @param {string} eventStart the start time of the event to create
+ * @param {string} eventEnd the end time of the event to create
+ */
+function createEvent(eventStart, eventEnd) {
+  const params = new URLSearchParams();
+  params.append('start', eventStart);
+  params.append('end', eventEnd);
+  fetch('/calendar', {method: 'POST', body: params});
+  populateCalendar();
+  populatePlanMail();
```

Comment:
would you still want to populate those panels if the fetch above failed?

Reply:
`Resolved`